libs/design/DateInput/DateInput.tsx

```diff
@@ -19,6 +19,11 @@ export const DateInput = forwardRef<HTMLInputElement, DateInputProps>(
         })}
         locale="fa"
         readOnly={readOnly}
+        rightSectionProps={{
+          style: {
+            display: readOnly ? 'none' : 'flex',
+          },
+        }}
```

Comment:
Could you explain shortly why do we need to do this?
cause we already said that if it is read-only, render null as the rightSection element

Reply:
Cause when the date input is empty the `rightSection` is there. 
what u did works just when the user fills the date input